azure-pipelines.yml

```diff
@@ -93,12 +93,6 @@ stages:
           GE_pytest_opts: '--no-sqlalchemy'
         strategy:
           matrix:
-            Python36-Pandas023:
```

Comment:
Do we also remove support for pandas 0.23 along with python 3.6?

Reply:
We shouldn't - this was preemptive of me.

Will follow the pattern you noted above (3.7 and Pandas 0.23)